src/screen/font.c

```diff
@@ -97,8 +97,10 @@ u8 open_font(const char* fontname)
     close_font();
 
     sprintf(filename, "media/%s.fon", fontname);
-    finit(&FontFAT, "media");
-    font.fh = fopen2(&FontFAT, filename, "rb");
+#ifdef FIL
```

Comment:
if FIL is not defined, how the line 19 get compiled?

Reply:
FIL is always either a define or a typedef.  I really need a better way to abstract this.  the petitfat and devofs use a single handle for filesystem and filehandle.  FatFS uses seperate

Reply:
is finit is not defined in FatFS? why not define a dummy to avoid this ifdef?